src/Fixer/SpecifyArgSeparatorFixer.php

```diff
@@ -0,0 +1,149 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\CodingStandard\Fixer;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+class SpecifyArgSeparatorFixer implements DefinedFixerInterface
+{
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Function `http_build_query()` should always have specified `$arg_separator` parameter',
+            [
+                new CodeSample("<?php\n\$queryString = http_build_query(['foo' => 'bar', 'baz' => 'bat']);"),
+            ],
+            'Function `http_build_query()` uses `arg_separator.output` ini directive as default argument separator, '
+            . 'however when its default value "&" is changed, query string assembled by the method will be '
+            . 'unexpectedly invalid. This Fixer forces you to not rely on ini settings and rather define '
+            . '`$arg_separator` in third parameter of the function.',
+            null,
+            null,
+            'Risky when other than default "&" argument separator should be used in query strings.'
+        );
+    }
+
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return $tokens->isTokenKindFound(T_STRING);
+    }
+
+    public function isRisky(): bool
+    {
+        return true;
+    }
+
+    public function fix(\SplFileInfo $file, Tokens $tokens): void
+    {
+        foreach ($tokens as $index => $token) {
+            if ($token->isGivenKind(T_STRING) && $token->getContent() === 'http_build_query') {
+                $previousIndex = $tokens->getPrevMeaningfulToken($index);
+
+                // Function imports and function definitions should not be affected
+                if ($previousIndex !== null
+                    && ($tokens[$previousIndex]->isGivenKind(CT::T_FUNCTION_IMPORT)
+                        || $tokens[$previousIndex]->isGivenKind(T_FUNCTION))) {
+                    return;
+                }
```

Comment:
I'd add all this to "shouldSkip()" method, it's not relevant to fixer process at all